### DISCUSSION TITLE
citar-template and csl json

### DISCUSSION THREAD

**Discussion by maasdorp** [emacs-citar/citar#547] [Q&A] [answered]
I use the default values for citar-template, but considering switching from biblatex to csl. The docs say:

> If you plan to use CSL JSON at all, you can and should include CSL JSON variables names where appropriate as such options. The default main template dates field demonstrates this.

Does this mean that I need to tinker with the default values, or that the default main template is sufficient for most use cases?

**Comment by bdarcus** [comment, marked as answer]
The latter.

It's a note for people that use CSL JSON and  want to modify the default templates.
